lib/src/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import styled from "styled-components";
+import TooltipPropsType from "./types";
+import * as Tooltip from "@radix-ui/react-tooltip";
+import CoreTokens from "../common/coreTokens";
+
+const DxcTooltip = ({ position = "bottom", label, children }: TooltipPropsType): JSX.Element => {
+  const SIZE = 8;
```

Comment:
Rename it to `size` I would prefer not to use a const for the size of the arrow.